admin-ui/plugins/auth-server/redux/api/OIDCApi.js

```diff
@@ -35,6 +35,14 @@ export default class OIDCApi {
     })
   }
 
+  getUMAResources = async (clientId) => {
```

Comment:
can we have this in separate UMAResourceApi.js file.

Reply:
done, updated @duttarnab